tests/cts/test_sparql_resolver/__init__.py

```diff
@@ -0,0 +1,108 @@
+from tests.cts.test_resolver import TestXMLFolderResolverBehindTheScene, TextXMLFolderResolver, TextXMLFolderResolverDispatcher
+from capitains_nautilus.cts.resolver import SparqlAlchemyNautilusCTSResolver, SleepyCatCTSResolver
+from capitains_nautilus.collections.sparql import generate_alchemy_graph, clear_graph, \
+    generate_sleepy_cat_graph
+from MyCapytain.common.constants import set_graph
+from ..config import sqlite_address, sleepy_cat_address
+from ...sparql_class import Sparql
+
+
+
+class _Parser(Sparql):
+    RESOLVER_CLASS = SparqlAlchemyNautilusCTSResolver
+
+
+class TestSparqlBasedResolverDispatcher(_Parser, TextXMLFolderResolverDispatcher):
```

Comment:
`TextXMLFolderResolverDispatcher` should probably be `TextXmlFolderResolverDispatcher`